utils/slsa_zero_two_converter.go

```diff
@@ -0,0 +1,124 @@
+package utils
```

Comment:
Could you move the `utils` folder under `/go` ?

Reply:
sure! sgtm!